internal/tiger/config/config.go

```diff
@@ -244,10 +250,10 @@ func setBool(key, val string) (bool, error) {
 	return b, nil
 }
 
-// updateField updates the field in the Config struct corresponding to the given key.
+// UpdateField updates the field in the Config struct corresponding to the given key.
 // It accepts either a string (from user input) or a typed value (string/bool from defaults).
 // The function validates the value and updates both the struct field and viper state.
-func (c *Config) updateField(key string, value any) (any, error) {
```

Comment:
Why did this function have to be exported? I'm not seeing where it's being used outside the package now 🤔